release-notes/major9/09.24.0/valtimo-backend-libraries.md

```diff
@@ -4,10 +4,10 @@
 
 The following features were added:
 
-* **New feature1**
+* **Search fields configuration deployment from resources**
 
-  Description of the new feature goes here.
-  Also link to the page that explains the feature in greater detail.
+  All search fields configurations that are placed in the resource folder, are now automatically deployed. More
```

Comment:
Can you change the first sentence to:

All search fields configurations placed in the resource folder are automatically deployed.